### ISSUE TITLE
add classnames for "rct-title" if checked or not 

### ISSUE THREAD

**Issue by VaheMartirosyan** [jakezatecky/react-checkbox-tree#367]
Hi there, I want to detect and change the design for halfchecked element titles #367

**Comment by jakezatecky** [comment]
If you want to change the half-check icon, you can do that by adding the `icons` property to the tree and supplying a value for `halfCheck`:

``` jsx
<CheckboxTree
    ...
    icons={{
        halfCheck: <WhateverJSX />,
    }}
/>
```

However, I assume you want to stylize the title text that is adjacent to a half-checked element? If so, the tree component does not flag the parent `label` state for easy CSS selection. **Nevertheless**, you may be able to use the [`has()` pseudo-class](https://drafts.csswg.org/selectors-4/#relational) to make a complicated selector that allows you to style the titles of nodes with the half-check state:

``` css
.rct-checkbox:has(> .rct-icon-half-check) + .rct-node-icon + .rct-title {
  /* Do whatever */
}
```

The `:has` selector has [limited browser support](https://caniuse.com/css-has), though.

**Comment by jakezatecky** [comment]
Closing due to lack of activity.
